Libraries/Image/RCTImageLoader.m

```diff
@@ -290,27 +290,44 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                            scale:(CGFloat)scale
                                                       resizeMode:(RCTResizeMode)resizeMode
                                                    progressBlock:(RCTImageLoaderProgressBlock)progressHandler
-                                                 completionBlock:(void (^)(NSError *error, id imageOrData))completionBlock
+                                                 completionBlock:(void (^)(NSError *error, id imageOrData, BOOL cacheResult))completionBlock
 {
   __block volatile uint32_t cancelled = 0;
   __block void(^cancelLoad)(void) = nil;
   __weak RCTImageLoader *weakSelf = self;
+  // Find suitable image URL loader
+  __block id<RCTImageURLLoader> loadHandler = [self imageURLLoaderForURL:imageURLRequest.URL];
+  __block BOOL cacheResult = [loadHandler respondsToSelector:@selector(shouldCacheLoadedImages)] ?
```

Comment:
You only need to mark variables as __block if you plan to re-assign them from inside a block. That doesn't seem to be the case here.